addon/styles/_frost-numeric-select.scss

```diff
@@ -0,0 +1,99 @@
+$height: 30px;
+$border: 0.3px solid $frost-color-input-border;
+$focus-border: 1px solid $frost-color-blue-1 !global;
+$frost-color-button-hover-bg: rgba($frost-color-blue-1, .1) !global;
+$arrow-focus-color: $frost-color-blue-1 !global;
+$error-text-color: $frost-color-input-error-border;
+$disabled-border: 1px solid $frost-color-input-disabled-border;
+$disabled-bg: $frost-color-input-disabled-bg;
+
+.frost-numeric-select {
+  position: relative;
+  display: block;
+  width: 70px;
+  height: $height;
+  margin-bottom: 15px;
+  outline: none;
+
+  &:focus {
+    border: $focus-border;
+  }
+
+  &:disabled {
+    border: $disabled-border;
+    background-color: $disabled-bg;
+  }
+}
+.frost-numeric-select-button {
+  position: relative;
+  display: block;
+  width: 70px;
+  height: $height;
+  margin-bottom: 15px;
+  outline: none;
+  padding-right: 26px;
+
+  &:focus {
+    border: $focus-border;
+  }
+}
+.error {
+  width: 175px;
+  height: $height;
+  font-size: 13px;
+  color: $error-text-color;
+}
+.container{
+  display: flex;
+  border: $border;
+  border-right: none;
+  border-top: none;
+  border-bottom: none;
+  flex-direction: column;
+  position: absolute;
+  right: 0px;
+  top: 0.7px;
+}
+.minusButton {
+  transform: rotate(180deg);
+  margin-top: 1px;
+  font-size: 14px;
+  height: 14px;
+  width: 25px;
+  padding: 0px 6.9px 0px 5.3px;
+  border-radius: 0px;
+  box-shadow: none;
+
+  &:hover {
+    background-color: $frost-color-button-hover-bg;
+  }
+
+  &:focus {
+    &:not(.disabled) {
+      outline: 0;
+      box-shadow: none;
+    }
+    .frost-icon-frost-numeric-chevron {
+      color: $arrow-focus-color;
+    }
+  }
+}
+.plusButton {
```

Comment:
Looks like it does: ```
  81:2  error  Class '.plusButton' should be written in lowercase with hyphens  class-name-format
```
Also, you have a lot of js lint and sass lint errors, which is why the build is failing. You can run: ```
$ lint-all-the-things
```
at the root of the `ember-frost-core` repo to see the list of all the errors. Ideally, you'll want to configure your editor to show linting errors, then you can just open the files referenced in the results of `lint-all-the-things` and fix the issues your editor shows in each one. Particularly for property order linting of sass, not having it displayed in your editor will make it much more painful to fix.